src/Microsoft.Extensions.HealthChecks/CompositeHealthCheckResult.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections.Generic;
+
+namespace Microsoft.Extensions.HealthChecks
+{
+    /// <summary>
+    /// Represents the results of multiple health checks
+    /// </summary>
+    public struct CompositeHealthCheckResult
```

Comment:
What's the value of this if it doesn't surface a combined status or some useful functionality? It's just a handle struct right now

Reply:
If this is a placeholder for maintainability, think about making it a class instead

Reply:
Yeah, it was a class in the previous sample, and it had a combined status value. I need to review exactly what our policy would be for generating combined status.

I plan to revisit this in the next iteration, so your feedback is noted :)